lib/errors/IdentitySTWrongVersionError.js

```diff
@@ -0,0 +1,40 @@
+const ConsensusError = require('./ConsensusError');
+
+class IdentitySTWrongVersionError extends ConsensusError {
+  /**
+   * @param {IdentityCreateStateTransition} stateTransition
+   * @param {number} currentVersion
+   */
+  constructor(stateTransition, currentVersion) {
+    super('Identity state transition version is too high');
+
+    this.name = this.constructor.name;
+    this.currentVersion = currentVersion;
+
+    this.stateTransition = stateTransition;
+
+    if (Error.captureStackTrace) {
```

Comment:
Why do you use  `captureStackTrace` and `this.name`, but not in other such errors?

Reply:
That's true, that doesn't make any sense since this code is already in ConsensusError

Reply:
Fixed